core/domain/user_jobs_one_off_test.py

```diff
@@ -1069,6 +1199,13 @@ def setUp(self):
 
         self.signup(self.OWNER_EMAIL, self.OWNER_USERNAME)
         self.owner_id = self.get_user_id_from_email(self.OWNER_EMAIL)
+        # Test reduce function does nothing.
+        self.assertIsNone(
+            user_jobs_one_off.UserProfilePictureOneOffJob.reduce(
+                'key', 'values'))
+
+    def mock_fetch_gravatar(self, unused_email):
```

Comment:
Prefix this with an underscore -- it should be private. (Also, good idea to streamline, thanks!)

Reply:
Done